README.md

```diff
@@ -75,6 +79,28 @@ To destroy stack just type:
 ``~ $ perun delete-stack <NAME OF YOUR STACK>
 ``
 
+You can use option ``--progress`` to show the stack creation/deletion progress in the console, but
+note, that this requires setting up a remote sink.
+##### Remote sink
```

Comment:
Please verify that it will render correctly, I'm pretty sure that lack of empty lines above and below header makes it rendered incorrectly.

Reply:
I did and it seems to render correctly, but I'll those empty lines anyway to make the md code more clear.